### ISSUE TITLE
Errors in loading the model and issues with the dataset

### ISSUE THREAD

**Issue by YAN-0802** [KWPCCC/MFAA#2]
Thank you for your work. I'm very interested in this idea, but after I downloaded the model weights according to the link, an error was reported:
`The Tensor name "InceptionV3 / AuxLogits/Conv2d_2a_3x3 / BatchNorm/beta" not found in checkpoint files ./models_tf/inception_v3.ckpt
[[node save/RestoreV2 (defined at MFAA.py:357)]]`
  Unfortunately, I couldn't find a good solution from the linked issue. How did you solve it? 

Also, is the 'labels.txt' correct? The results I ran using them were quite different from the original text, but I didn't make major changes to the code. However, I suspect it might be a problem with the label file or the dataset folder. Because the label in the first line of the 1.png file does not correspond to that in the label.txt file. 

Finally, the environment you are using is TensorFlow 1.12.2. I think it is also possible that the environment configuration is not defined, and it requires CUDA 9.0 configuration. What type of graphics card are you using in this environment? I tried to configure it on the 2080Ti and 3090, but it doesn't support the use of CUDA 9.0.

Thank you again for your contributions, and I look forward to hearing from you.
